lib/paperclip/attachment.rb

```diff
@@ -33,7 +33,7 @@ def self.default_options
         :use_timestamp         => true,
         :whiny                 => Paperclip.options[:whiny] || Paperclip.options[:whiny_thumbnails],
         :validate_media_type   => true,
-        :adapter_options       => { hash_digest: Digest::MD5 },
+        :adapter_options       => { },
```

Comment:
This hash contains all the default options. For documentation purposes, I'd leave the default here, and not in code as it was.